rules/test/snapshot/BazelRunfileBridge.mm

```diff
@@ -0,0 +1,23 @@
+#import "BazelRunfileBridge.h"
+#include "tools/cpp/runfiles/runfiles.h"
+#include <string>
+
+using namespace bazel::tools::cpp::runfiles;
+
+NSString *BazelRunfilePathForResource(NSString *resource)
+{
+    // TODO: confirm if Bazel runfile support will work E2E 👍
```

Comment:
There's still todos in here?

Reply:
Yeah this PR was listed as scaffolding to get a function call to the referenced library wired up and ensure it's buildable and callable at a basic level. Weather or not the lib is a complete answer for snapshots loading needs more testing and consideration and you can atleast more easily try it out. These are left as TODOs here that we can remove once https://github.com/bazel-ios/rules_ios/issues/563 is resolved.